### DISCUSSION TITLE
不懂就问，关注满了，天选时刻还能继续抽吗

### DISCUSSION THREAD

**Discussion by Only-Mio** [andywang425/BLTH#285] [Q&A] [answered]
关注满了继续运行，发现要求关注的天选时刻显示参与了，但没关注up，这样能中奖吗？

**Comment by Only-Mio** [comment, marked as answer]
还有那个转盘抽奖能用吗？一直提示网络错误，求解，谢谢

**Reply by andywang425** [reply]
转盘的数据在gitee上，但是由于gitee的相关政策这个数据获取不到，而且数据的提供者已经很久没更新转盘数据了。

**Reply by Only-Mio** [reply]
好的，谢谢！那关于最前面说的那个天选时刻问题，需不需要清关注？

**Reply by andywang425** [reply]
> 好的，谢谢！那关于最前面说的那个天选时刻问题，需不需要清关注？

这个我不清楚

**Reply by Only-Mio** [reply]
好的感谢
